cmd/cluster/core/create_test.go

```diff
@@ -218,3 +249,120 @@ func TestValidate(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
Insead of adding this new test can we consolidate with https://github.com/openshift/hypershift/pull/5775/files#diff-23216e73706c06453a939915d3547af12db4de806b4a1a7d81552269a539d5a6
i.e.
- can we please include validation for this field input and cover it within TestValidate

- Can we also add it to TestPrototypeResources

- Can we also include a case in aws/azure to validate the rendered resource against a fixture